packages/components/src/internal/components/editable/actions.ts

```diff
@@ -422,19 +422,21 @@ export async function addRows(
  * @param queryColumns the ordered map of columns to be added
  * @param fieldKey the fieldKey of the existing column after which the new columns should be inserted.  If undefined
  * or the column is not found, columns will be added at the beginning.
+ * @param readOnlyColumns set of columns that are in the grid but may be designated not as showInInsertView (e.g., 'Name')
```

Comment:
nit: I believe it's `shownInInsertView`. This comment seems a little disjoint. Why does adding columns care about `shownInInsertView`? Seems like this comment may be better off being a little more general.